cornell-dti/campus-density-android
# Day chips fixes

### Summary 

This PR works towards implementing changes to chips on the `FacilityInfoPage` that we discussed in our meeting.

- Moved Day Chips above the Historical Data Card
- Enabled Day Chips to change menu on facility info page
- Made Menu Card only show when there is a menu to show
- Removed some of the hard-coded string values for days and replaced with resource strings
- Added styles for chips (`BaseChipStyle` and `OutlineChipStyle`)
- Fixed spacing and code throughout facility info page

### Test Plan 
- Use Postman to determine what menu should be for future day in a week and check that the app is showing the correct menu when the day is switched using the day chips.
- Try opening Cafe Jennie Facility Info page and ensure that the Menu Card does not show at all 



![88064002_128314965287832_2745381934992457728_n](https://user-images.githubusercontent.com/8313666/75633010-78b97780-5bcf-11ea-92f2-30619c92961a.png)
![87881251_535560843738117_3658431457442922496_n](https://user-images.githubusercontent.com/8313666/75633012-7a833b00-5bcf-11ea-81d5-a40eb49e5872.png)
